src/opencl/blake2b.cl

```diff
@@ -301,40 +257,34 @@ static int blake2b_update( blake2b_state *S, const uchar *in, ulong inlen )
 	  inlen -= inlen;
 	}
   }
-
   return 0;
 }
-
-static int blake2b_final( blake2b_state *S, uchar *out, uchar outlen )
+/* Is this correct? */
+static int blake2b_final( __private blake2b_state *S, uchar *out, uchar outlen )
```

Comment:
What's this comment about? AFAIK it's correct.

Reply:
Oops, sorry about that, the comment was copy-pasted from Nano, removed